distribution/ddf/pom.xml

```diff
@@ -17,17 +17,16 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <artifactId>kernel</artifactId>
-        <groupId>org.codice.ddf</groupId>
+        <artifactId>distribution</artifactId>
+        <groupId>ddf</groupId>
         <version>2.11.0-SNAPSHOT</version>
-        <relativePath>../kernel/pom.xml</relativePath>
     </parent>
     <!--
     Packages the DDF Standard distribution, which consists of the DDF Core distribution.​ 
 
     Note that this distribution has the DDF Kernel distribution as its parent, hence all of the maven goals in the DDF Kernel distribution pom will execute unless they are overridden in this pom.
     -->
-    <artifactId>ddf</artifactId>
+    <artifactId>ddf-build</artifactId>
```

Comment:
Instead of changing the artifact ID to `ddf-build`, I think you should set the group ID to `org.codice.ddf` and keep the artifact ID as `ddf`.  This will keep the POM and the ZIP together where they have been.

http://artifacts.codice.org/content/groups/public/org/codice/ddf/ddf/